src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
+import countInputValidation from "./utils/countInputValidation.js";
+import progress from "./utils/progress.js";
+import racePrint from "./utils/racePrint.js";
+import winner from "./utils/winner.js";
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(MESSAGES.START);
+    const CARS_NAMES = await MissionUtils.Console.readLineAsync("");
+    carInputValidation(CARS_NAMES);
+
+    MissionUtils.Console.print(MESSAGES.ANSWER_COUNT);
+    const COUNT = await MissionUtils.Console.readLineAsync("");
+    countInputValidation(COUNT);
+
+    //게임 진행
+    MissionUtils.Console.print("\n" + MESSAGES.RESULT);
+    const CARS = CARS_NAMES.split(",");
+
+    const RACE = {};
+    CARS.forEach((CAR) => {
+      RACE[CAR.trim()] = 0;
+    });
+
+    //레이싱 결과 출력
+    for (let i = 0; i < COUNT; i++) {
+      Object.entries(RACE).forEach(([CAR, PROGRESS]) => {
+        if (progress()) {
+          RACE[CAR]++;
+        }
+      });
```

Comment:
```suggestion
      Object.entries(RACE).forEach(([CAR, _]) => {
        if (progress()) {
          RACE[CAR]++;
        }
      });
```
`[CAR, PROGRESS]`에서 `PROGRESS`는 사용하지 않고, 아래 함수 이름과도 겹치는 부분이 있어서 `_`으로 변경해서 사용하지 않는다는 의도를 더 명확히 표현해줘도 좋을 것 같아요~!

Reply:
생략 가능 한 걸 까먹고 있었내요 좋은 지적 감사합니다 ~